### PR TITLE
Do no longer force-install a default nginx.conf

### PR DESCRIPTION
**Pull request by SonicHedgehog**
As far as I can tell, a default `nginx.conf` does no longer need to be force-installed. Let me know if I forgot something.

### CODE REVIEW

File: plugins/nginx-vhosts/install [hunk 1]
```diff
@@ -22,45 +22,12 @@ ssl_session_cache shared:SSL:20m;
 ssl_session_timeout 10m;
 
 ssl_ciphers EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;
-ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
-ssl_prefer_server_ciphers on;
```
Comment (josegonzalez) [inline plugins/nginx-vhosts/install]: Why did you remove these ciphers?

**Comment by josegonzalez** [comment]
The force install was introduced [here](https://github.com/progrium/dokku/issues/799).

File: plugins/nginx-vhosts/install (hunk 1 above)
Comment (SonicHedgehog) [inline plugins/nginx-vhosts/install]: I have removed the `ssl_protocols` and `ssl_prefer_server_ciphers` directives because they are already included in the default `nginx.conf` of nginx 1.8.0 (the current stable version). Otherwise, nginx would throw an `[emerg] directive is duplicate` error.

Comment (josegonzalez) [inline plugins/nginx-vhosts/install]: Is that version installed with ubuntu 14.04?

Comment (SonicHedgehog) [inline plugins/nginx-vhosts/install]: No, but [it is installed when installing dokku](https://github.com/progrium/dokku/blob/v0.3.22/plugins/nginx-vhosts/dependencies#L9) as it uses the [PPA nginx/stable](https://launchpad.net/~nginx/+archive/ubuntu/stable).

Comment (josegonzalez) [inline plugins/nginx-vhosts/install]: Gotcha, makes sense to me.

**Comment by josegonzalez** [comment]
Seems like tests fail?

**Comment by SonicHedgehog** [comment]
Fixed.

File: plugins/nginx-vhosts/install (hunk 1 above)
Comment (josegonzalez) [inline plugins/nginx-vhosts/install]: If you install dokku from source, it won't use the PPA. Hmmm...

**Comment by josegonzalez** [comment]
Going to close this. A real fix needs to take into account the fact that we don't use the PPA for nginx with the debian solution :( I'll take care of this in #1467.

**Comment by josegonzalez** [comment]
Scratch that. Seems this can go in after all :)

**Comment by josegonzalez** [comment]
Closing in favor of #1468. Thanks for your work @SonicHedgehog!

**Comment by michaelshobbs** [comment]
Whoa! We want both I think, yea?

**Comment by josegonzalez** [comment]
Right I'm a nub. Reopened.
